### PR TITLE
feat: New AWS Gov Cloud Cfg resource

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Aws GovCloud Config Resource

Signed-off-by: Darren Murray <darren.murray@lacework.net>

### CODE REVIEW

File: website/docs/r/integration_aws_gov_cloud_ct.html.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# lacework\_integration\_aws\_gov\_cloud\_ct
+
+Use this resource to configure an AWS GovCloud CloudTrail integration analyze CloudTrail activity for monitoring cloud account security.
+
+To find more information see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360021140214-Initial-Setup-of-AWS-GovCloud-Integration).
+
+## Example Usage
+
+```hcl
+resource "lacework_integration_aws_gov_cloud_cfg" "example" {
```
Comment (dmurray-lacework) [inline website/docs/r/integration_aws_gov_cloud_ct.html.markdown:18]:
```suggestion
resource "lacework_integration_aws_gov_cloud_ct" "example" {
```

File: website/docs/r/integration_aws_gov_cloud_cfg.html.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# lacework\_integration\_aws\_gov\_cloud\_cfg
+
+Use this resource to configure an AWS GovCloud Config integration to analyze AWS configuration compliance.
+
+To find more information see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360021140214-Initial-Setup-of-AWS-GovCloud-Integration).
+
+## Example Usage
+
+```hcl
+resource "lacework_integration_aws_gov_cloud_cfg" "example" {
```
Comment (dmurray-lacework) [inline website/docs/r/integration_aws_gov_cloud_cfg.html.markdown:18]: Rename to lacework_integration_aws_govcloud_cfg ?

File: website/docs/r/integration_aws_gov_cloud_ct.html.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Use this resource to configure an AWS GovCloud CloudTrail integration analyze CloudTrail activity for monitoring cloud account security.
+
+To find more information see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360021140214-Initial-Setup-of-AWS-GovCloud-Integration).
+
+## Example Usage
+
+```hcl
+resource "lacework_integration_aws_gov_cloud_ct" "example" {
+	name      = "AWS gov cloud config integration example"
```
Comment (dmurray-lacework) [inline website/docs/r/integration_aws_gov_cloud_ct.html.markdown:19]:
```suggestion
	name      = "AWS gov cloud cloudtrail integration example"
```

File: website/docs/r/integration_aws_gov_cloud_cfg.html.markdown (hunk 2 above)
Comment (afiune) [reply website/docs/r/integration_aws_gov_cloud_cfg.html.markdown:18]: Definitely! 💯

**Pull request closed without merging**